docs/generated/packages/nx.json

```diff
@@ -0,0 +1,345 @@
+{
```

Comment:
I'm not sure this file should have been added. Can you `rm -rf docs && yarn documentation` on your next rebase?